bin/node-gyp.js

```diff
@@ -25,14 +25,18 @@ prog.parseArgv(process.argv)
 prog.devDir = prog.opts.devdir
 
 var homeDir = osenv.home()
-if (prog.devDir) {
+if (process.env.LOCALAPPDATA) {
+  // TODO: create intermediate directories
```

Comment:
What is the best way to do this in node without any dependencies?  I found http://stackoverflow.com/a/40249973/879046, but I'm not sure if that is appropriate in this case or not.